cmd/openshift-install/gather.go

```diff
@@ -162,6 +162,10 @@ func logGatherBootstrap(bootstrap string, port int, masters []string, directory
 	if err := ssh.PullFileTo(client, fmt.Sprintf("/home/core/log-bundle-%s.tar.gz", gatherID), file); err != nil {
 		return errors.Wrap(err, "failed to pull log file from remote")
 	}
+	apiServerFiles := filepath.Join(directory, fmt.Sprintf("log-bundle-api-server-%s.tar.gz", gatherID))
```

Comment:
Let's not pull two separate bundles. That is going to be confusing to the user. Let's find a way to include the most recent results from the api server in the other log bundle.

1. When the cluster-bootstrap pods exits with an error code, gather the cluster resources. As the last part of gathering resources, check if the openapi.json.gz file is of sufficient size. If it is, then store the cluster resources in a known location.
2. When installer-gather.sh is executed, do the same gathering of cluster resources, including moving to a known location if appropriate. Then, copy the cluster resources from the known location to the bundle location, so that they will be included in the bundle.